src/components/Modals/ClaimNameFatFingerModal/ClaimNameFatFingerModal.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { ModalNavigation, Field, Button } from 'decentraland-ui'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import { t, T } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Props, State } from './ClaimNameFatFingerModal.types'
+import './ClaimNameFatFingerModal.css'
+
+export default class ClaimNameFatFingerModal extends React.PureComponent<Props, State> {
+  state = {
```

Comment:
this needs to be `state: State = {` in order to enforce the typing, otherwise you can put whatever you want in that `state` and typescript won't complain